.github/workflows/release-elixir.yaml

```diff
@@ -18,6 +18,8 @@ jobs:
       - name: Install extism shared library
         shell: bash
         run: |
+          mkdir -p /home/runner/.local/bin/  
+          export PATH="/home/runner/.local/bin/ :$PATH"
```

Comment:
Does the extra space bewteen `/bin/` and `:$PATH` exist in the resulting variable? Worried that would cause an issue.

Reply:
no i'll fix though